bin/oref0-setup.sh

```diff
@@ -452,7 +452,7 @@ else
     openaps device add pump mmeowlink subg_rfspy $ttyport $serial $radio_locale || die "Can't add pump"
     openaps alias add wait-for-silence '! bash -c "(mmeowlink-any-pump-comms.py --port '$ttyport' --wait-for 1 | grep -q comms && echo -n Radio ok, || openaps mmtune) && echo -n \" Listening: \"; for i in $(seq 1 100); do echo -n .; mmeowlink-any-pump-comms.py --port '$ttyport' --wait-for 30 2>/dev/null | egrep -v subg | egrep No && break; done"'
     openaps alias add wait-for-long-silence '! bash -c "echo -n \"Listening: \"; for i in $(seq 1 200); do echo -n .; mmeowlink-any-pump-comms.py --port '$ttyport' --wait-for 45 2>/dev/null | egrep -v subg | egrep No && break; done"'
-    if [[ ${radio_locale,,} =~ "WW" ]]; then
+    if [[ "$radio_locale" =~ "WW" ]]; then
```

Comment:
The convention I've been using thus far is to lowercase everything for comparison:

```
scott@scott-mbp ~/oref0 $ grep ,, bin/oref0-setup.sh
if ! [[ ${CGM,,} =~ "g4" || ${CGM,,} =~ "g5" || ${CGM,,} =~ "mdt" || ${CGM,,} =~ "shareble" || ${CGM,,} =~ "xdrip" ]]; then
    if [[ ${CGM,,} =~ "shareble" ]]; then
if [[ ${CGM,,} =~ "shareble" ]]; then
if [[ ${CGM,,} =~ "shareble" ]]; then
if [[ ${CGM,,} =~ "xdrip" ]]; then
if [[ ! -z "$BT_MAC" || ${CGM,,} =~ "shareble" ]]; then
if [[ ${CGM,,} =~ "g5" ]]; then
elif [[ ${CGM,,} =~ "shareble" ]]; then
    if [[ ${radio_locale,,} =~ "WW" ]]; then
if [[ ${CGM,,} =~ "mdt" ]]; then
if [[ ${CGM,,} =~ "xdrip" ]]; then
    if [[ ${CGM,,} =~ "shareble" ]]; then
    elif [[ ${CGM,,} =~ "xdrip" ]]; then
    elif ! [[ ${CGM,,} =~ "mdt" ]]; then # use nightscout for cgm
if [[ ${CGM,,} =~ "shareble" ]]; then
```